### ISSUE TITLE
Pagination: links are not wrapped in `li`

### ISSUE THREAD

**Issue by mroach** [woylie/flop_phoenix#278]
I'm getting Flop pagination setup with my Tailwind theming. I noticed an issue that when I had an ellipsis showing up, my horizontal list broke.

Looking into the HTML generated by Flop, I noticed something odd: inside the `ul` for the pagination items, only the ellipsis element is wrapped in an `li` and the rest are `a` tags directly under the `ul`. This means the resulting HTML looks like:

```html
<ul class="pagination-list">
  <a>1</a>
  <a>2</a>
  <li>
    <span>...</span>
  </ii>
  <a>99</a>
</ul>
```

https://github.com/woylie/flop_phoenix/blob/12e6c15d5d44243c9959f40774b242f7cdc419f7/lib/flop_phoenix.ex#L502-L504

Part of my problem is not being able to apply classes to the `li` that I need to get it to render properly.
Aside from this, it seems not complaint with the [HTML standard for `ul`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul#technical_summary) to have non-`li` elements as children.

**Comment by woylie** [comment]
Thanks for opening the issue. This is bad, sorry about that. I just released 0.22.2 with a fix.

**Comment by mroach** [comment]
Thanks so much for the quick fix! :)
